Shared/UI/Views/MessageContentView/MessageContentView.swift

```diff
@@ -353,8 +353,12 @@ class MessageContentView: BaseView {
                     self.emotionLabel.alpha = 0.0
                 }
                 
-                self.addEmotionImageView.alpha = 1.0
-                self.addEmotionButton.alpha = 1.0
+                // Only allow the author to add emotions
+                if let msg = self.message, msg.isFromCurrentUser {
```

Comment:
See comment. Maybe we re introduce this for everyone when we have a way display who the emotion is from.